src/Microsoft.Framework.CopyOnWriteDictionary.Internal/project.json

```diff
@@ -0,0 +1,15 @@
+{
+    "version": "1.0.0-*",
+    "shared": "*.cs",
+    "dependencies": {
```

Comment:
I was actually wondering about the opposite issue: how does `kpm pack` or `kpm build` in this directory _succeed_ without the dependency?